src/meinberlin/meinberlin/tests/acceptance/setup_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
On the detail page bash is not always in '/bin', it's often in /usr/bin/bash.

It would be better to use a Python script for the setup as it allows more powerful things to be done in the setup and reduce the dependencies on the bash or sh interpreter.
 
We also need one setup_tests (with 's'!) for each buildable frontend module, otherwise protractor will encounter an error trying to execute a script that is not there.